src/pages/Designer/Designer.js

```diff
@@ -25,35 +25,111 @@ const Designer = () => (
-                margin={{
-                  top: 'none',
-                  bottom: 'medium',
-                  left: 'large',
-                }}
+      <ResponsiveContext.Consumer>
+        {(responsive) =>
+          responsive === 'large' || responsive === 'xlarge' ? (
+            <Box>
+              <Box height="medium" margin={{ top: 'xlarge' }}>
+                <Stack margin={{ top: 'xlarge' }} anchor="left">
+                  <Box
+                    margin={{ top: 'xlarge' }}
+                    height="gradient1"
+                    background="gradient"
+                  />
+                  <Box margin={{ top: 'large' }} direction="row">
+                    <IconCard
+                      size="medium"
+                      icon={
+                        <ClearOption size="xxlarge" color="DesignerYellow" />
+                      }
+                      margin={{ left: 'xlarge', top: 'large' }}
+                    />
+                    <Box
+                      direction="column"
+                      margin={{
+                        left: 'xlarge',
+                        top: 'xlarge',
+                        bottom: 'medium',
+                      }}
+                    >
+                      <Text> Grommet Tools </Text>
+                      <Heading level={1} size="xlarge" margin={{ top: 'none' }}>
+                        Designer
+                      </Heading>
+                      <Paragraph
+                        margin={{ top: 'none' }}
+                        size="xxlarge"
+                        color="darkGrey"
+                      >
+                        Grommet Designer is a tool used to build expiriences
+                        with Grommet components; then publish and share your
+                        ideas with a simple wysiwg interface
+                      </Paragraph>
+                      <Box
+                        round="medium"
+                        background="buttonYellow"
+                        onClick={() => {}}
+                        width="small"
+                        height="xxsmall"
+                        justify="center"
+                        margin={{ bottom: 'small' }}
+                      >
+                        <Text weight="bold" alignSelf="center" color="black">
+                          Open Designer
+                        </Text>
+                      </Box>
+                    </Box>
+                  </Box>
+                </Stack>
+              </Box>
+              <Box
+                direction="column"
+                gap="medium"
+                margin={{ left: 'xlarge', bottom: 'large' }}
               >
-                Designer
-              </Heading>
+                <Box direction="row" margin={{ left: 'xlarge' }} gap="xlarge">
+                  <Camera opacity="0.8" size="large" color="ImagerOrange" />
+                  <Configure opacity="0.9" size="large" color="ThemerOrange" />
```

Comment:
Icon doesn't have an `opacity` prop.

Reply:
I wasn't sure how else to change the opacity for the icons

Reply:
Did it actually work? do you see errors on the console?

Reply:
Yeah it worked and no errors on the console

Reply:
I'll need to take a look at it later.